internal/backend/runtime/omni/controllers/omni/cluster_machine_encryption.go

```diff
@@ -27,7 +27,9 @@ import (
 var encryptionConfigPatchData string
 
 // ClusterMachineEncryptionController manages disk encryption configuration for the cluster.
-type ClusterMachineEncryptionController struct{}
+type ClusterMachineEncryptionController struct {
+	CompressConfigData bool
```

Comment:
is it used?

Reply:
nope, seems I left some traces from my previous approach where I explicitly passed this to the accessors. will clean all of them up.